### PR TITLE
DOC: Add example for rescaling/resizing/downscaling

### LINKED ISSUE #2559: Add example in gallery about downscaling/upscaling/resizing images
In the gallery, it would be interesting to add an example with the following functions:

* http://scikit-image.org/docs/stable/api/skimage.transform.html#downscale-local-mean
* http://scikit-image.org/docs/stable/api/skimage.transform.html#rescale
* http://scikit-image.org/docs/stable/api/skimage.transform.html#resize

If anyone is interested to make this contribution, that would be very useful I guess.

Could be added in the section "Geometrical transformations and registration"

### PR DESCRIPTION
**Pull request by souravsingh**
## Description
Closes #2559 

## Checklist
[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
- [x] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Unit tests

[For detailed information on these and other aspects see [scikit-image contribution guidelines](http://scikit-image.org/docs/dev/contribute.html)]

## References
[If this is a bug-fix or enhancement, it closes issue # ]
[If this is a new feature, it implements the following paper: ]

### CODE REVIEW

**Comment by souravsingh** [comment]
This would be the first PR for the Docathon.

File: doc/examples/transform/plot_rescale.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+"""
+=======
+Rescale
+=======
+
+The Rescale operation performs interpolation to
```
Comment (stefanv) [inline doc/examples/transform/plot_rescale.py]: Wrap to 80.

I think we need to be careful about the language here.  Rescaling is seen in some communities as "rescaling of intensity".  So, perhaps something like:

"The rescale operation resizes an image by a given scaling factor.  The scaling factor can either be a single floating point value, or multiple values---one along each axis."

I think `interpolation=None` is now the default in matplotlib.

And yay for the first docathon commit!

File: doc/examples/transform/plot_rescale.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+"""
+=======
+Rescale
+=======
+The rescale operation resizes an image by a given scaling factor.
+The scaling factor can either be a single floating point value,
+or multiple values---one along each axis.
+
+The Example takes a camera image as input and rescales it by a
```
Comment (stefanv) [inline doc/examples/transform/plot_rescale.py]: This example takes the 'camera' image and rescales it by...

**Comment by jni** [comment]
I have two comments in addition to @stefanv's suggested fix:

- Currently, this plots the original as a very zoomed-in version, and the new image in full size. I think it would be more intuitive if the original was full-size and the new version appeared smaller. Is there a way to make that happen? Maybe using ax.set_ylim/set_xlim?
- It might be interesting to compare this function to `resize`, which takes in the target shape, instead of the rescaling factor, in this same example.

**Comment by sciunto** [comment]
I also think that we can discuss the three functions mentioned in the issue in the same example. They are closely related.

**Comment by souravsingh** [comment]
Comparing the three functions in a single example is a good idea.

**Comment by souravsingh** [comment]
I have added the three functions in the example as a comparison.

**Comment by sciunto** [comment]
@souravsingh Could you tick the box 'allow modifications from maintainers' (on the right). I'll make some edits.

Thanks!

**Comment by souravsingh** [comment]
@sciunto It is already ticked.

**Comment by sciunto** [comment]
Thanks. I made the changes. @stefanv or any other @scikit-image/core  for a double-check?

**Comment by soupault** [comment]
How about this version?
rescale

**Comment by sciunto** [comment]
even better.

**Comment by jni** [comment]
@souravsingh now you need to add an `=` under the headline. =)

**Comment by soupault** [comment]
Hmm... Has anyone checked if the rendering is correct? I'd expect some issues - https://github.com/scikit-image/scikit-image/pull/2560#commitcomment-21681650

**Comment by jni** [comment]
Oops! You are completely correct. I misread the order of changes/approvals. I'll submit a fix now.

**Comment by soupault** [comment]
Anyways, thanks @souravsingh for pushing this! Good work!

**Pull request merged**